optuna/cli.py

```diff
@@ -20,30 +20,26 @@
 from cliff.commandmanager import CommandManager
 from cliff.lister import Lister
 
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
```

Comment:
Could you place these imports from `typing` with the other standard library imports at the top (after `import types`)?

Reply:
Thank you for your review. I fixed code.